web_transport/sdk/api/owt/quic/web_transport_server_interface.h

```diff
@@ -4,26 +4,26 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef OWT_QUIC_QUIC_TRANSPORT_SERVER_INTERFACE_H_
-#define OWT_QUIC_QUIC_TRANSPORT_SERVER_INTERFACE_H_
+#ifndef OWT_WEB_TRANSPORT_WEB_TRANSPORT_SERVER_INTERFACE_H_
+#define OWT_WEB_TRANSPORT_WEB_TRANSPORT_SERVER_INTERFACE_H_
 
 #include "owt/quic/export.h"
-#include "owt/quic/quic_transport_session_interface.h"
+#include "owt/quic/web_transport_session_interface.h"
 
 namespace owt {
 namespace quic {
-// A server accepts WebTransport - QuicTransport connections.
-class OWT_EXPORT QuicTransportServerInterface {
+// A server accepts WebTransport connections.
+class OWT_EXPORT WebTransportServerInterface {
  public:
   class Visitor {
    public:
     virtual ~Visitor() = default;
     // Called when server is ended.
```

Comment:
Please specify the semantics of "ended"?

Reply:
Server is stopped. All connections are closed and no new connection will be accepted.